MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -225,15 +225,31 @@
 
                                 @foreach (var ingredient in Model.Recipe.Ingredients)
                                 {
+                                    @*<li class="recipe__item">
+                    <svg class="recipe__icon">
+                        <use href="img/icons.svg#icon-check"></use>
+                    </svg>
+                    <div class="recipe__count">2</div>
+                    <div class="recipe__ingredient">
+                        <span class="recipe__unit">g</span>
+                        pizza
```

Comment:
pizza?

Reply:
This is all commented out. Part of the previously hard code in the original solution